src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useEffect } from 'react';
```

Comment:
there is a console error for this file, see screenshot:

<img width="1321" alt="Screenshot 2021-02-10 at 10 09 52" src="https://user-images.githubusercontent.com/65414651/107488781-1fe8b200-6b88-11eb-8865-50ef92079d28.png">

Can you fix it? We should be looking out for those errors and fixing them during development -- they can potentially break production in unexpected ways, and are also a pain for local development and testing

Reply:
this one occurs when I am on the data integration step and when I click to the second elbow plot view

Reply:
Changes to remove this one is related to the plot styling config. There will be another ticket for the plot styling config, which should remove this,